src/Simulator/IncrementalRedstoneSimulator/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required (VERSION 2.6)
+project (MCServer)
+
+include_directories ("${PROJECT_SOURCE_DIR}/../")
+
+set (SRCS
+	RedstoneHandler.cpp
+	IncrementalRedstoneSimulator.cpp
+	RedstoneHandlerFactory.cpp
+)
+
+set (HDRS
+	CommandBlockHandler.h
+	DoorHandler.h
+	DropSpenserHandler.h
+	IncrementalRedstoneSimulator.h
+	RedstoneHandler.h
+	RedstoneSimulatorChunkData.h
+	RedstoneHandlerFactory.h
+	SolidBlockHandler.h
+	RedstoneComparatorHandler.h
+	RedstoneRepeaterHandler.h
+	RedstoneBlockHandler.h
+	RedstoneTorchHandler.h
+	RedstoneWireHandler.h
+	RedstoneLampHandler.h
+	RedstoneToggleHandler.h
+	PistonHandler.h
+	SmallGateHandler.h
+	NoteBlockHandler.h
+	TNTHandler.h
+	TrappedChestHandler.h
+	TripwireHookHandler.h
+	PoweredRailHandler.h
+	PressurePlateHandler.h
+)
```

Comment:
They're still needed though, not least so that everything appears in Visual Studio's Solution Explorer.